android/filament-android/src/main/java/com/google/android/filament/Engine.java

```diff
@@ -239,6 +239,18 @@ public Builder featureLevel(FeatureLevel featureLevel) {
             return this;
         }
 
+        /**
+         * Sets the initial paused state of the rendering thread.
```

Comment:
Here we need to document the caveats of this API:
- "buffer" callbacks won't be called until unpaused. So, in particular, if the user is relying on the callbacks to unpause, that would create a deadlock that would be undetectable by us.
- if the user is calling too many APIs that fill up the queue, it will eventually block and since we're not draining it, this will also deadlock. But we can detect that one and assert.